searchcore/src/vespa/searchcore/proton/server/proton.cpp

```diff
@@ -310,8 +310,11 @@ Proton::init(const BootstrapConfig::SP & configSnapshot)
         protonConfig.basedir.c_str(), getcwd(tmp, sizeof(tmp)));
 
     _persistenceEngine = std::make_unique<PersistenceEngine>(*this, _diskMemUsageSampler->writeFilter(),
+                                                             _diskMemUsageSampler->notifier(),
                                                              protonConfig.visit.defaultserializedsize,
                                                              protonConfig.visit.ignoremaxbytes);
+    _diskMemUsageSampler->notifier().addDiskMemUsageListener(&_persistenceEngine->get_resource_usage_tracker());
```

Comment:
Is this needed? It seems to already happen in the constructor of ResourceUsageTracker, which the PersistenceEngine instance owns.